### PR TITLE
Support for `ABORTMSG` and `ASSERTMSG`

### LINKED ISSUE #954: [Neo v3.6.0] Add new opcode `ABORTMSG`
https://github.com/neo-project/neo-vm/commit/7e5996844a90b514739f879bc9f873f9a34c9a67

### LINKED ISSUE #955: [Neo v3.6.0] Add new opcode `ASSERTMSG`
https://github.com/neo-project/neo-vm/commit/7e5996844a90b514739f879bc9f873f9a34c9a67

### PR DESCRIPTION
**Pull request by mialbu**
Closes #954 
Closes #955

### CODE REVIEW

**Comment by mialbu** [comment]
Integration tests are passing locally.

File: int-tests/src/test-integration/java/io/neow3j/compiler/HelperIntegrationTest.java [hunk 1]
```diff
@@ -337,6 +347,12 @@ public static void abort(boolean bool) {
             }
         }
 
+        public static void abortMsg(boolean bool) {
+            if (bool) {
```
Comment (csmuller) [inline int-tests/src/test-integration/java/io/neow3j/compiler/HelperIntegrationTest.java:351]: It's the little things, like this check... 👏💪🫡

**Review by csmuller — APPROVED** [review]
Great! 😃

**Comment by csmuller** [comment]
The Codacy issues do not apply because they are from Java smart contract in the tests.

**Pull request merged**